crates/compiler/solve/tests/solve_expr.rs

```diff
@@ -1469,6 +1469,21 @@ mod solve_expr {
         infer_eq("{ x: 5, y : 3.14 }.x", "Num *");
     }
 
+    #[test]
+    fn record_literal_accessor_function() {
+        infer_eq(".x { x: 5, y : 3.14 }", "Num *");
+    }
+
+    #[test]
+    fn tuple_literal_accessor() {
+        infer_eq("(5, 3.14 ).0", "Num *");
+    }
+
+    #[test]
+    fn tuple_literal_accessor_function() {
+        infer_eq(".0 (5, 3.14 )", "Num *");
+    }
+
```

Comment:
can some tests for the type of just tuples and just tuple accessors be added?

Reply:
Added a test for tuple types (that's now pushed).

Then I realized you also asked about tuple accessor - which it turns out don't work off the bat. Need to resolve todo!() :)

Reply:
Ok, done!